src/app/system/manage-hooks/manage-hooks.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Hook
+  </button>
+</div>
+
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Hook Template </th>
+        <td mat-cell *matCellDef="let hook"> {{ hook.name }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="displayName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Hook Name </th>
+        <td mat-cell *matCellDef="let hook"> {{ hook.displayName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="isActive">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let hook"> {{ hook.isActive }} </td>
```

Comment:
@abhaychawla done, but looking at the chart of accounts, isn't it better to call it `status` instead of `disabled`? That suggests that every field is under `disabled` category.

Reply:
Fixed to active/inactive :)